### PR TITLE
feat(modules): add coverage options for modules

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

File: generators/module/index.js [hunk 1]
```diff
@@ -49,16 +49,23 @@ module.exports = class extends Generator {
       type: 'input',
       name: 'userName',
       message: 'Your name',
-      default: username.substring(0, username.length - 1)
+      default: username.substring(0, username.length - 1),
+      store: true
```
Comment (targos) [inline generators/module/index.js]: please no. you should store your .yorc in your home directory, not in the project

File: generators/module/index.js [hunk 2]
```diff
@@ -49,16 +49,23 @@ module.exports = class extends Generator {
...
     }, {
       type: 'input',
       name: 'email',
       message: 'Your email',
-      default: email.substring(0, email.length - 1)
+      default: email.substring(0, email.length - 1),
+      store: true
```
Comment (targos) [inline generators/module/index.js]: same

File: generators/module/templates/package [hunk 3]
```diff
@@ -13,7 +13,8 @@
     "eslint-fix": "npm run eslint -- --fix",
     "prepublish": "rollup -c",
     "test": "run-s testonly eslint",
-    "testonly": "jest"
+    <% if (codecov) { %>"test-travis": "jest --coverage && codecov",
```
Comment (targos) [inline generators/module/templates/package]: put this at the end of the preceding line to avoid having a blank line when the option is false

Comment (targos) [inline generators/module/templates/package]: you should also run the eslint script on travis

File: generators/module/templates/package [hunk 4]
```diff
@@ -12,9 +12,9 @@
     "eslint": "eslint src",
     "eslint-fix": "npm run eslint -- --fix",
     "prepublish": "rollup -c",
-    "test": "run-s testonly eslint",
-    <% if (codecov) { %>"test-travis": "jest --coverage && codecov",
-    <% } %>"testonly": "jest"
+    "test": "run-s testonly eslint",<% if (codecov) { %>
+    "test-travis": "eslint src && jest --coverage && codecov",<% } %>
```
Comment (targos) [inline generators/module/templates/package]: if I'm not mistaken, the `<% } %>` should still be on the next line

Comment (maasencioh) [inline generators/module/templates/package]: I think that it will merge the two lines

**Pull request merged**